web/src/features/moreCast2/components/GroupHeader.tsx

```diff
@@ -1,14 +1,14 @@
 import { Checkbox, FormControlLabel, FormGroup, IconButton, Popover, Stack, Typography, styled } from '@mui/material'
 import { ExpandMore } from '@mui/icons-material'
 import React, { ChangeEvent, MouseEvent, useState } from 'react'
-import { LIGHT_GREY, MEDIUM_GREY, DARK_GREY } from 'app/theme'
+import { LIGHT_GREY, MEDIUM_GREY, DARK_GREY, MoreCastParamColors } from 'app/theme'
 import { ColumnVis } from 'features/moreCast2/components/DataGridColumns'
 
 interface GroupHeaderProps {
   id: string
   columns: ColumnVis[]
-  weatherParam: string
-  handleShowHideChange: (weatherParam: string, columnName: string, value: boolean) => void
+  weatherParam: keyof MoreCastParamColors
+  handleShowHideChange: (weatherParam: keyof MoreCastParamColors, columnName: string, value: boolean) => void
```

Comment:
We should update the type of the `weatherParam` parameter in `handleShowHideChange()` in `TabbedDataGrid` as well.